xinference/model/rerank/core.py

```diff
@@ -112,29 +112,52 @@ def __init__(
         self._device = device
         self._model_config = model_config or dict()
         self._use_fp16 = use_fp16
+        self._use_sentence_transformers = model_spec.model_name in [
```

Comment:
How about for model that type == 'normal' or not specified type just using sentence_transformers? If user register rerank, what library will use?

Reply:
Good point. Currently, the most popular rerank model is the normal type.